doc/scripts/genrest/genrest.py

```diff
@@ -84,9 +48,12 @@ def print_items(items, outdir, indent):
 
 The configuration options' information below is extracted directly from
 :program:`Kconfig` using the :file:`~/doc/scripts/genrest/genrest.py` script.
-Click on the option name in the table below for detailed information about
-each option.
+Click on the symbol name in the table below for to see its definition.
```

Comment:
Heh... not sure why I changed this. Leftover from some earlier rewriting I think. I could change it back.